frontend/src/components/allocations.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react';
+import { NomadLink } from './link';
+import DisplayTime from './time'
+import ReactTooltip from 'react-tooltip'
+
+class AllocationList extends Component {
```

Comment:
I personally prefer explicit names, the file should have been `allocation_list` actually :)